tests/integration-tests/configs/develop.yaml

```diff
@@ -198,8 +198,7 @@ test-suites:
       dimensions:
         - regions: ["us-east-1"]
           instances: ["c5n.18xlarge"]
-          # TODO: use INSTANCES_DEFAULT_X86 once there is centos8 support for efa
-          oss: ["alinux", "alinux2", "centos7", "ubuntu1604", "ubuntu1804"]
+          oss: {{ common.INSTANCES_DEFAULT_X86 }}
```

Comment:
I think there is an error here cause you are using instances and not oss :)

Reply:
Fixed